wallets/rn_cli_wallet/src/screens/SubscriptionSettingsScreen.tsx

```diff
@@ -0,0 +1,197 @@
+import * as React from 'react';
+import {useNavigation, useRoute} from '@react-navigation/native';
+import {
+  ActivityIndicator,
+  Alert,
+  PlatformColor,
+  Pressable,
+  ScrollView,
+  StyleSheet,
+  Switch,
+  Text,
+  View,
+} from 'react-native';
+import useNotifyClientContext from '@/hooks/useNotifyClientContext';
+
+import {Controller, useForm} from 'react-hook-form';
+import {useSafeAreaInsets} from 'react-native-safe-area-context';
+import {useBottomTabBarHeight} from '@react-navigation/bottom-tabs';
+import {useTheme} from '@/hooks/useTheme';
+
+type BooleanMap = {[key: string]: boolean};
+
+export default function SubscriptionSettingsScreen() {
+  const {params} = useRoute();
+  const {bottom} = useSafeAreaInsets();
+  const tabBarHeight = useBottomTabBarHeight();
+  const topic = params?.topic;
+  const {navigate} = useNavigation();
+
+  const Theme = useTheme();
+  const {subscriptions, notifyClient} = useNotifyClientContext();
+  const [unsubscribing, setUnsubscribing] = React.useState(false);
+
+  const subscription = subscriptions.find(s => s.topic === topic);
+  const scopes = subscription?.scope || {};
+
+  let scopesBooleanMap: BooleanMap = {};
+  Object.keys(scopes).map((key: string) => {
+    scopesBooleanMap[key] = scopes[key].enabled;
+  });
+  const notificationTypes = Object.keys(subscription?.scope || {}).map(
+    key => scopes[key],
+  );
+
+  const {control, watch} = useForm({
+    defaultValues: scopesBooleanMap,
+  });
+
+  async function handleSaveNotificationSettings() {
+    const newScopes = watch();
+    const enabledScopes = Object.keys(newScopes).filter(
+      key => newScopes[key] === true,
+    );
+
+    if (!notifyClient) {
+      Alert.alert('Notify client not initialized');
+      return;
+    }
+
+    const updated = await notifyClient.update({
+      topic,
+      scope: enabledScopes,
+    });
+  }
+
+  async function handleUnsubscribe() {
+    if (!notifyClient) {
+      Alert.alert('Notify client not initialized');
+      return;
+    }
+    setUnsubscribing(true);
+
+    await notifyClient
+      .deleteSubscription({topic})
+      .then(() => {
+        setUnsubscribing(false);
+        navigate('SubscriptionsScreen');
+      })
+      .catch(e => {
+        Alert.alert('Failed to unsubscribe', e);
+        setUnsubscribing(false);
+      });
+  }
+
+  return (
+    <ScrollView
+      contentInsetAdjustmentBehavior="automatic"
+      contentContainerStyle={{
+        paddingHorizontal: 16,
+      }}>
+      <View
+        style={{
+          width: '100%',
```

Comment:
move inline styles to styles object, keep the `backgroundColor: Theme['gray-glass-005']` only